channel/consensus_channel/consensus_channel_test.go

```diff
@@ -67,16 +67,53 @@ func TestConsensusChannel(t *testing.T) {
 		)
 
 	}
+
+	fingerprint := func(v Vars) string {
+		h, err := v.asState(state.TestState.FixedPart()).Hash()
+
+		if err != nil {
+			panic(err)
+		}
+		return h.String()
+	}
+
+	vars := Vars{TurnNum: 9, Outcome: outcome()}
+
+	f1 := fingerprint(vars)
+	clone1 := vars.clone()
+
+	if fingerprint(clone1) != f1 {
+		t.Fatal("vars incorrectly cloned: ", f1, fingerprint(clone1))
+	}
+
+	clone1.Outcome.guarantees[targetChannel] = guarantee(999, bob, bob, bob)
+	if f1 != fingerprint(vars) {
+		t.Fatal("vars shares data with clone")
+	}
+
+	clone2 := vars.clone()
```

Comment:
This test is ineffective -- see https://github.com/statechannels/go-nitro/pull/411/commits/4612ae86e3935197c15f500a708dab0d42a612a1